Classes/common/CDTFetchChanges.m

```diff
@@ -91,7 +100,10 @@ - (SequenceNumber)notifyChanges:(TD_RevisionList *)changes
     }
```

Comment:
Addressed in f5222af. See note in commit as to why I didn't add a cancelled check within the loop over the doc array returned from `getDocumentsWithIds`.